lib/mongo/monitoring/event/cmap/connection_closed.rb

```diff
@@ -0,0 +1,102 @@
+# Copyright (C) 2016-2019  MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  class Monitoring
+    module Event
+      module Cmap
+
+        # Event published when a connection is closed.
+        #
+        # @since 2.8.0
+        class ConnectionClosed < Base
+
+          # @return [ Symbol ] STALE Indicates that the connection was closed due to it being stale.
+          #
+          # @since 2.8.0
+          STALE = :stale
+
+          # @return [ Symbol ] IDLE Indicates that the connection was closed due to it being idle.
+          #
+          # @since 2.8.0
+          IDLE = :idle
```

Comment:
Also, we do not need any of these constants, the symbols can be returned directly.

Reply:
This is not idiomatic Ruby. If you want to keep the constants, that is fine, but their values still need to be appropriate for someone to use directly.

Reply:
I'm not sure what you're suggesting I change here; I've updated the `POOL_CLOSED` constant to `:pool_closed`, and I don't see any casing issues with the others. Can you be more specific about what here needs to be updated?

Reply:
No action is needed, I am just pointing out the fact that most applications will not be using these constants.